src/pyproject.toml

```diff
@@ -0,0 +1,15 @@
+[tool.poetry]
```

Comment:
Since we're not using Poetry, this file should be deleted.

Reply:
Misunderstood the note about the explanation file, I changed the  paragraph, not the header. I have now changed the header as requested.